x-pack/plugins/fleet/common/constants/routes.ts

```diff
@@ -29,6 +29,7 @@ export const EPM_API_ROUTES = {
   DELETE_PATTERN: EPM_PACKAGES_ONE,
   FILEPATH_PATTERN: `${EPM_PACKAGES_FILE}/{filePath*}`,
   CATEGORIES_PATTERN: `${EPM_API_ROOT}/categories`,
+  SUMMARY_PATTERN: `${EPM_PACKAGES_MANY}/{pkgName}/summary`,
```

Comment:
nitpick maybe change the name to something a little less generic `/stats` `/usages`

Reply:
👍  Let's go with `/stats` since @jen-huang also suggested that for the name